localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
-d_orcish_shamanism;The Farseers;Les Longs-voyants;Die Scharfseher;;;;;;;;;;;x
-d_orcish_shamanism_adj;Shamanic;;schamanisch;;;;;;;;;;;x
-d_shamanism;The Shamanic Council;Le Conseil chamanique;Der schamanische Rat;;;;;;;;;;;x
-d_shamanism_adj;Shamanic;;schamanisch;;;;;;;;;;;x
-d_earth_mother_worship;The Earth-Temple;Le Temple de la Terre;Tempel der Erde;;;;;;;;;;;x
-d_earth_mother_worship_adj;Earthen;;irdisch;;;;;;;;;;;x
-d_tidemother;The Tidesages;;Die Gezeitenweisen;;;;;;;;;;;x
-d_tidemother_adj;Tidesage;;;;;;;;;;;;;x
-d_odyn;The Halls of All-Father;;Die Hallen des Allvaters;;;;;;;;;;;x
-d_odyn_adj;Odyn's;;odynisch;;;;;;;;;;;x
-d_helya;The Temple of the Sea-Witch;Le Temple de la Sorci�re des mers;Tempel der Seehexe;;;;;;;;;;;x
-d_helya_adj;Helya's;;helyasisch;;;;;;;;;;;x
+d_primitive_shamanism;The Sacred Woods;Les Bois Sacrés;Die heiligen Wälder;;El bosque Sagrado;;;;;;;;;x
+d_primitive_shamanism_adj;Shamanic;Chamanique;schamanisch;;Chamánico;;;;;;;;;x
+d_orcish_shamanism;The Farseers;Les Longs-voyants;Die Scharfseher;;Los Clarividentes;;;;;;;;;x
+d_orcish_shamanism_adj;Shamanic;Chamanique;schamanisch;;Chamánico;;;;;;;;;x
+d_shamanism;The Shamanic Council;Le Conseil chamanique;Der schamanische Rat;;El Consejo Chamánico;;;;;;;;;x
+d_shamanism_adj;Shamanic;Chamanique;schamanisch;;Chamánico;;;;;;;;;x
+d_earth_mother_worship;The Earth-Temple;Le Temple de la Terre;Tempel der Erde;;El Templo de la Tierra;;;;;;;;;x
+d_earth_mother_worship_adj;Earthen;En Terre;irdisch;;De barro;;;;;;;;;x
+d_tidemother;The Tidesages;Des Eaugures;Der Gezeitenweisen;;Los Sabiomar;;;;;;;;;x
+d_tidemother_adj;Tidesage;Eaugure;gezeitenweise;;Sabiomar;;;;;;;;;x
+d_odyn;The High Halls of the All-Father;Les salles exaltées du Père-tout-puissant;Den heiligen Hallen des allmächtigen Vaters;;La Cámara Alta del Gran Padre;;;;;;;;;x
+d_odyn_adj;Odyn's;d’Odyn;odyns;;de Odyn;;;;;;;;;x
+d_helya;The Temple of the Sea-Witch;Le Temple de la Sorcière des mers;Tempel der Seehexe;;;;;;;;;;;x
+d_helya_adj;Helya's;De Helya;helyas;;de Helya;;;;;;;;;x
 d_voidfarer;The Walkers;Les Marcheurs;;;;;;;;;;;;x
-d_voidfarer_adj;Void;;;;;;;;;;;;;x
-d_forsaken_cult;The Cult of Forgotten Shadows;Le Culte des Ombres oubli�es;Kult des vergessenen Schattens;;;;;;;;;;;x
-d_forsaken_cult_adj;Shadow;Ombre-oubli�e;;;;;;;;;;;;x
+d_voidfarer_adj;Void;Vide;;;Vacío;;;;;;;;;x
+d_forsaken_cult;The Cult of Forgotten Shadows;Le Culte des Ombres oubliées;Kult des vergessenen Schattens;;;;;;;;;;;x
+d_forsaken_cult_adj;Shadow;Ombre-oubliée;;;;;;;;;;;;x
 PROV132;Helm;Helm;;;;;;;;;;;;x
-d_lightbound;The Lightbound Crusaders;Les Crois�s Sancteforges;Die lichtgebundenen Kreuzfahrer;;;;;;;;;;;x
-d_lightbound_adj;Lightbound;;lichtgebunden;;;;;;;;;;;x
-d_drustwar;Drustvar;;;;;;;;;;;;;x
+d_lightbound;The Lightbound;Empreint de Lumière;Der Lichtgebundenen;;Vinculada a la Luz;;;;;;;;;x
+d_lightbound_adj;Lightbound;Empreint de Lumière;lichtgebundener;;vinculado a la Luz;;;;;;;;;x

```

Comment:
lichtgebunden

Reply:
This came from here: https://de.wowhead.com/item=151570/lichtgebundener-kristall

Is that not correct? Since that's an adjective to the crystal word?

Reply:
It would be right, but as I explained with the Sentinels already, the ending gets added by ck2 already. So you just go with "lichtgebunden" and "e" or "er" gets added by the game, depending on what word comes next.

Reply:
Gotcha.